src/app/app.module.ts

```diff
@@ -85,6 +87,10 @@ export function initialize(
       provide: HTTP_INTERCEPTORS,
       useClass: HttpErrorInterceptor,
       multi: true
+    },
+    {
+      provide: RECAPTCHA_V3_SITE_KEY,
+      useValue: '6LfBmGojAAAAAGawo276q4BH1gOfxi96ugYNq3aE'
     }
```

Comment:
This value should not be hardcoded. We should obtain the recaptcha public key from the ConfigService. You can do that by changing the block to
```
    {
      provide: RECAPTCHA_V3_SITE_KEY,
      useFactory: (configService: ConfigService) => {
        return configService.getRecaptchaPublicKey();
      },
      deps: [ConfigService]
    }
```